recipes/conda-forge-pinning/conda_build_config.yaml

```diff
@@ -0,0 +1,215 @@
+blas_impl:
+  - mkl                        # [x86 or x86_64]
+  - openblas
```

Comment:
Could you please elaborate a bit more about how this works and how it is intended be used?

Reply:
See https://github.com/AnacondaRecipes/numpy-feedstock/blob/master/recipe/meta.yaml